### PR TITLE
output/eve: add 'verdict' field to 'alert' and 'drop' events - v5

### PR DESCRIPTION
**Pull request by jufajardini**
Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) ticket:
https://redmine.openinfosecfoundation.org/issues/5464

Previous PR: https://github.com/OISF/suricata/pull/8542

Changes from last PR:
- optimize changes, based on feedback on previous PR

suricata-verify-pr: 1124
https://github.com/OISF/suricata-verify/pull/1124

### CODE REVIEW

**Comment by suricata-qa** [comment]
WARNING:
 
 | field | baseline | test | % |
|--|--|--|--| 
| | SURI_TLPW1_stats_chk | | |
| .tcp.rst | 113528 | 145844 | 128.47% |
| | SURI_TLPR1_stats_chk | | |
| .flow.memuse | 521782368 | 847523264 | 162.43% | 

Pipeline 12576

File: src/output-json-alert.c [hunk 1]
```diff
@@ -1059,6 +1079,13 @@ static void JsonAlertLogSetupMetadata(AlertJsonOutputCtx *json_output_ctx,
         json_output_ctx->payload_buffer_size = payload_buffer_size;
     }
 
+    /* action is enabled by default */
+    flags |= LOG_JSON_ACTION;
+    SetFlag(conf, "action", LOG_JSON_ACTION, &flags);
```
Comment (jufajardini) [inline src/output-json-alert.c:1084]: I think that this change is triggering the debug validation checks. Investigating....

Comment (jufajardini) [reply src/output-json-alert.c:1084]: It is that, indeed, as conf can be NULL here.

**Comment by jufajardini** [comment]
Fixed in https://github.com/OISF/suricata/pull/8559

**Pull request closed without merging**